imap_processing/hi/packet_definitions/TLM_HI_COMBINED_SCI.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version='1.0' encoding='utf-8'?>
 <xtce:SpaceSystem xmlns:xtce="http://www.omg.org/space/xtce" name="H45">
-	<xtce:Header date="2024-11-21 00:00:00" version="1" author="IMAP SDC" />
+	<xtce:Header date="2024-11-26 00:00:00" version="1" author="IMAP SDC" source_file="TLM_H45_H90_20241126.xlsx" />
```

Comment:
nice! I think we all should store source file

Reply:
I can put that in a PR change to `excel_to_xtce`.